src/pages/harm.js

```diff
@@ -1,10 +1,84 @@
-import React from 'react'
-import Link from 'gatsby-link'
+import React from 'react';
+import Link from 'gatsby-link';
+import Img from 'gatsby-image';
 
-const HarmPage = () => (
+import alternativesIcon from '../assets/images/alternatives-icon.png';
+import consequencesIcon from '../assets/images/consequences-icon.png';
+
+import styles from './harm.module.scss';
+import sharedStyles from '../assets/shared-styles/shared.module.scss';
+
+const HarmPage = ({ data }) => (
   <div>
-    <h1>Harm</h1>
+    <Img
+      alt="Text overlay that reads: At the end of my first day, my legs and arms were covered in bruises."
+      sizes={data.harmImage.sizes}
+    />
+
+    <div className={sharedStyles.pageWrapper}>
+      <div>
+        <p className={styles.firstStatement}>Beatings, rapes and murder are a reality of prostitution.</p>
+        <p className={styles.firstStatement}>You dont want to buy in to that.</p>
+      </div>
+      <div className={sharedStyles.dividerLine}></div>
+      <div>
+        <h1 className={styles.statPercentage}>85% to 95%</h1>
+        <p className={styles.statCaption}>of those in prostitution want to escape it, but have no other options for survival</p>
+      </div>
+      <div className={sharedStyles.dividerLine}></div>
+      <div>
+        <h1 className={styles.statPercentage}>60% to 75%</h1>
```

Comment:
nice reuse of the `statPercentage` and `statCaption` classes on all the parts that look the same!